lib/app.js

```diff
@@ -71,6 +71,7 @@ export default class TrimetWebapp extends Component {
     /** the main webapp **/
     return (
       <ResponsiveWebapp
+        customIcons={customIcons}
```

Comment:
It feels a bit weird to be passing the `customIcons` here AND as a prop to `desktopView`. Just wondering if you attempted any other approaches/refactors.

Reply:
Yeah, I really don't like sending the props like this. Maybe there's a way to put them in the redux store.

Reply:
@evansiroky to create a ticket for a customIcons refactor.